integration_test/third_party_apps_data/applications/mongodb/enable

```diff
@@ -11,6 +11,7 @@ metrics:
   receivers:
     mongodb:
       type: mongodb
+      insecure: true
```

Comment:
What is the implication / reason behind this change?

Reply:
Since i authored the PR, I'll chime in here. 

There was an initial bug in the mongodb receiver where TLS settings were getting ignored in some scenarios, more details can be found in this PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/12093.

This meant that all usages of the mongodb receiver previously were using insecure client side connections which most likely would have gotten rejected by the mongodb server and preventing metrics from being collected. 

With the fix everything is updated to use proper TLS that the user supplies.

Reply:
Got it. Thanks for the clarification!